Forbidden: /api/v1/dslamport/events/
Forbidden: /api/v1/dslam/events/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/switch-command/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router-command/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/switch/read_switch_backup_error_files_name/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/switch/read_switch_backup_error_files_name/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router-command/read_router_backup_error_files_name/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Forbidden: /api/v1/dslam/events/
Forbidden: /api/v1/dslamport/events/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Forbidden: /api/v1/dslam/events/
Forbidden: /api/v1/dslamport/events/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/download_router_backup_file/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router-command/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/router/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/router/get_router_backup_files_name/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/dslam/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/users/login/
Unauthorized: /api/v1/users/login/
Internal Server Error: /api/v1/router-command/router_run_command/
Internal Server Error: /api/v1/router-command/router_run_command/
Unauthorized: /api/v1/router-command/router_run_command/
Unauthorized: /api/v1/router-command/router_run_command/
Unauthorized: /api/v1/dslam/
Unauthorized: /api/v1/switch/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/get_router_backup_files_name/
Unauthorized: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/dslam/get_dslams_packet_loss_count/
Unauthorized: /api/v1/dslamport/events/
Unauthorized: /api/v1/dslam/events/
Unauthorized: /api/v1/dslam/
Unauthorized: /api/v1/dslam-port/
Unauthorized: /api/v1/dslam/
Unauthorized: /api/v1/dslam/
Unauthorized: /api/v1/dslam/
Internal Server Error: /api/v1/dslam/icmp/command/
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/exception.py", line 47, in inner
    response = get_response(request)
  File "/usr/local/lib/python3.8/dist-packages/django/core/handlers/base.py", line 179, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python3.8/dist-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/django/views/generic/base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python3.8/dist-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python3.8/dist-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.8/dist-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.8/dist-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/portmanv3/portman_web/dslam/views.py", line 1166, in post
    result = utility.run_icmp_command(dslam_id, icmp_type, params)
  File "/opt/portmanv3/portman_web/dslam/utility.py", line 61, in run_icmp_command
    return c.run_icmp_command(dslam_id, icmp_type, params)
  File "/usr/local/lib/python3.8/dist-packages/jsonrpclib/jsonrpc.py", line 821, in __call__
    return self.__send(self.__name, args)
  File "/usr/local/lib/python3.8/dist-packages/jsonrpclib/jsonrpc.py", line 695, in _request
    response = self._run_request(request)
  File "/usr/local/lib/python3.8/dist-packages/jsonrpclib/jsonrpc.py", line 730, in _run_request
    response = self.__transport.request(
  File "/usr/lib/python3.8/xmlrpc/client.py", line 1153, in request
    return self.single_request(host, handler, request_body, verbose)
  File "/usr/local/lib/python3.8/dist-packages/jsonrpclib/jsonrpc.py", line 413, in single_request
    self.send_content(connection, request_body)
  File "/usr/local/lib/python3.8/dist-packages/jsonrpclib/jsonrpc.py", line 476, in send_content
    connection.endheaders()
  File "/usr/lib/python3.8/http/client.py", line 1247, in endheaders
    self._send_output(message_body, encode_chunked=encode_chunked)
  File "/usr/lib/python3.8/http/client.py", line 1007, in _send_output
    self.send(msg)
  File "/usr/lib/python3.8/http/client.py", line 947, in send
    self.connect()
  File "/usr/lib/python3.8/http/client.py", line 918, in connect
    self.sock = self._create_connection(
  File "/usr/lib/python3.8/socket.py", line 808, in create_connection
    raise err
  File "/usr/lib/python3.8/socket.py", line 796, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Forbidden: /api/v1/dslam/events/
Forbidden: /api/v1/dslamport/events/
Internal Server Error: /api/v1/dslam/get_interface_traffic_input/
Unauthorized: /api/v1/router/
Unauthorized: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
Not Found: /api/v1/router/
